### DISCUSSION TITLE
Ipad performance

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#981] [General]
Started to notice the my iPad (2 years old) started to work very slow for some reason.
Takes ages to load something on it.
Any tips how to make it working smooth again?

**Comment by hernessteve** [comment]
Just clear all the cache and cookies.
That should help

**Comment by MirkosMangle** [comment]
There are a few simple methods to achieve that, nothing too complicated. I won't delve into extensive details about optimizing iPads for speed, as you can easily find comprehensive guides on the [macgasm](https://www.macgasm.net/ios/ipad-running-slow/) website. Their articles cover various tips and tricks to enhance the performance of your iPad, ensuring you get the most out of your device without unnecessary complexity. Simply head over to their website and explore the wealth of information available to optimize your iPad experience.
